website/content/docs/secrets/databases/elasticdb.mdx

```diff
@@ -173,6 +176,32 @@ the proper permission, it can generate credentials.
    username           v-vaultuser-my-role-AgIViC5TdQHBdeiCxae0-1602541724
    ```
 
+### Static Roles
+
+Static roles return the same credentials for every request. The credentials are rotated based on the rotation period provided.
```

Comment:
> rotated based on the rotation period provided

Maybe we should make this more generic since we have period and schedule-based rotations after 1.15?

Reply:
That's a good point, would `based on the schedule provided` work to encapsulate both?

Reply:
Sounds great!